src/examples/java/io/nats/examples/stability/JsMulti.java

```diff
@@ -0,0 +1,506 @@
+// Copyright 2015-2018 The NATS Authors
```

Comment:
Hey, if you added some measurements (msgs/sec and bytes/sec) this would be a very nice benchmark...

Reply:
In the last PR I added to NatsAutobench the following JsPubSync JsPubAsync JsSub JsPubRounds (Sawtooth)
This was intended to be a runner, I can add timings here and / or I can add more to the AutoBench

Reply:
Ok I added in stats
```
Thread 1   |      400,000 msgs |      337.885 ms |  1,183,836 msg/s |  144.51 mb/s
Thread 2   |      400,000 msgs |      334.872 ms |  1,194,486 msg/s |  145.81 mb/s
Thread 3   |      400,000 msgs |      329.524 ms |  1,213,873 msg/s |  148.18 mb/s
Total      |    1,200,000 msgs |     1,002.28 ms |  1,197,270 msg/s |  146.15 mb/s
```